hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -127,6 +126,10 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       status.getMessage(),
       liveState.getStagingTasksSize()));
 
```

Comment:
Cool, thanks. I think this check can be (liveState.getCurrentAcquisitionPhase() == AcquisitionPhase.RECONCILING_TASKS) . We might put the other phase correction check before this one also.